### PR TITLE
Add region to sample table

### PR DESCRIPTION
**Pull request by ttung**
### Description
Added a migration that manually writes "North America" to all samples.  I'm not aware of any biohub samples that were from outside of North America, and even if there were, there are certainly no plans to import them into aspen.

Depends on #235 

### Test plan
ran the migration

### CODE REVIEW

File: src/py/aspen/database/models/sample.py [hunk 1]
```diff
@@ -123,6 +123,11 @@ class Sample(idbase, DictMixin):
             }
         },
     )
+    region = Column(
+        String,
```
Comment (phoenixAja) [inline src/py/aspen/database/models/sample.py]: would it make more sense to make this an enum field instead?

Comment (ttung) [reply src/py/aspen/database/models/sample.py]: I looked at the region data for all the gisaid data and it appears very constrained to the continents.  I think it's pretty safe to make it an enum.

File: src/py/aspen/database/models/accessions.py [hunk 2]
```diff
@@ -51,6 +51,7 @@ def add_to_entity(
         workflow_start_datetime: Optional[datetime.datetime] = None,
         workflow_end_datetime: Optional[datetime.datetime] = None,
     ):
+        """"""
```
Comment (phoenixAja) [inline src/py/aspen/database/models/accessions.py]: rm?

**Review by phoenixAja — APPROVED** [review]
looks good!!

**Pull request merged**